### DISCUSSION TITLE
Slow Response Time for First Prompt When Using Certain Models (e.g., Llama3.1)

### DISCUSSION THREAD

**Discussion by fatmaT2001** [open-webui/open-webui#6187] [General]
**Description**:
Hello OpenWebUI team,

I’ve noticed that certain models, such as **Llama3.1**, experience a significant delay in responding to the first prompt, particularly during the GPU connection phase. This issue doesn't seem to affect all models, as others connect and respond much faster during the initial interaction.

### Details:
- **Models experiencing delay**: Llama3.1 (and other similar models)
- **Issue**: First prompt response time is considerably slower while connecting to the GPU, but subsequent prompts respond faster.
- **Behavior**: The delay occurs only during the initial prompt, and the system performs normally afterward.

### Question:
Could you please provide insight into why certain models, like Llama3.1, take longer to respond during the first prompt while connecting to the GPU, whereas other models don’t exhibit this behavior? Is there a difference in model initialization or other factors causing this delay?

Thank you for your assistance in helping me understand this behavior!

**Comment by sameleff** [comment]
Have you resolved this? I'm noticing this same issue, though I haven't yet found a model that responds quickly to the first prompt. Do you have an example of some that responded quickly for you so I can test?
If I watch the system load via Task Manager, I can tell that it takes a very long time for ollama to load the model into VRAM...as soon as the resource usage increases to the expected value, the model outputs the response.
